hivemq-edge/src/frontend/src/extensions/datahub/components/controls/Toolbox.tsx

```diff
@@ -48,34 +50,43 @@ export const Toolbox = () => {
             aria-label={t('workspace.toolbox.aria-label') as string}
             backgroundColor="var(--chakra-colors-chakra-body-bg)"
           >
-            <VStack pl={2} alignItems="flex-start">
-              <Text id="group-pipeline">{t('workspace.toolbox.group.pipeline')}</Text>
-              <ButtonGroup variant="outline" size="sm" aria-labelledby="group-pipeline">
-                <Tool nodeType={DataHubNodeType.TOPIC_FILTER} />
-                <Tool nodeType={DataHubNodeType.CLIENT_FILTER} />
-              </ButtonGroup>
-            </VStack>
-            <VStack alignItems="flex-start">
-              <Text id="group-dataPolicy">{t('workspace.toolbox.group.dataPolicy')}</Text>
-              <ButtonGroup variant="outline" size="sm" aria-labelledby="group-dataPolicy">
-                <Tool nodeType={DataHubNodeType.DATA_POLICY} />
-                <Tool nodeType={DataHubNodeType.VALIDATOR} />
-                <Tool nodeType={DataHubNodeType.SCHEMA} />{' '}
-              </ButtonGroup>
-            </VStack>
-            <VStack alignItems="flex-start">
-              <Text id="group-behaviorPolicy">{t('workspace.toolbox.group.behaviorPolicy')}</Text>
-              <ButtonGroup variant="outline" size="sm" aria-labelledby="group-behaviorPolicy">
-                <Tool nodeType={DataHubNodeType.BEHAVIOR_POLICY} />
-                <Tool nodeType={DataHubNodeType.TRANSITION} />
-              </ButtonGroup>
-            </VStack>
-            <VStack alignItems="flex-start" pr={2}>
-              <Text id="group-action">{t('workspace.toolbox.group.action')}</Text>
-              <ButtonGroup variant="outline" size="sm" aria-labelledby="group-action">
-                <Tool nodeType={DataHubNodeType.OPERATION} />
-              </ButtonGroup>
-            </VStack>
+            <ButtonGroup variant="outline" size="sm" aria-labelledby="group-pipeline">
+              <VStack pl={2} alignItems="flex-start">
+                <Text id="group-pipeline">{t('workspace.toolbox.group.pipeline')}</Text>
+                <HStack>
+                  <Tool nodeType={DataHubNodeType.TOPIC_FILTER} />
+                  <Tool nodeType={DataHubNodeType.CLIENT_FILTER} />
+                </HStack>
+              </VStack>
+            </ButtonGroup>
+            <ButtonGroup variant="outline" size="sm" aria-labelledby="group-dataPolicy">
+              <VStack alignItems="flex-start">
+                <Text id="group-dataPolicy">{t('workspace.toolbox.group.dataPolicy')}</Text>
+                <HStack>
+                  <Tool nodeType={DataHubNodeType.DATA_POLICY} />
+                  <Tool nodeType={DataHubNodeType.VALIDATOR} />
+                  <Tool nodeType={DataHubNodeType.SCHEMA} />{' '}
```

Comment:
suggestion: empty space might not be needed

```suggestion
                  <Tool nodeType={DataHubNodeType.SCHEMA} />
```